pkg/httpclient/client.go

```diff
@@ -89,6 +89,13 @@ func (c *Client) SetHeader(key string, value string) {
 	c.headers[key] = value
 }
 
+func (c *Client) SetQueryParam(key, value string) {
+	if c.queryParams == nil {
+		c.queryParams = make(map[string]string)
+	}
+	c.queryParams[key] = value
+}
```

Comment:
I'm not particularly happy with this implementation, but I wanted to do as little changes as possible given this is likely a temporary feature of Vulture.

Reply:
👍 This approach to set global query params fine to me.